README.md

```diff
@@ -40,7 +40,7 @@ ALLOWED_HOSTS=*
 CORS_ORIGIN_ALLOW_ALL=1
 NOTIFICATIONS_ENABLED=1
 
-VENE_PAYMENTS_BAMBORA_API_URL=https://fake-bambora-api-url/api
+VENE_PAYMENTS_BAMBORA_API_URL=https://real-bambora-api-url/api
```

Comment:
Any reason to change this?

Reply:
Some payment tests fail when running locally if this setting is used

Reply:
I think https://fake-bambora-api-url/api is meant to be used to trigger a simulated failure with bambora

Reply:
AHHH right, good point.